tests/core/conftest.py

```diff
@@ -42,6 +42,10 @@
 
 E2E_STORY_FILE_UNKNOWN_ENTITY = "data/test_evaluations/story_unknown_entity.md"
 
+TRIGGERS_AND_FORMS_DOMAIN = "data/test_domains/triggers_and_forms.yml"
+NO_FORMS_POLICY_CONFIG = "data/test_config/no_form_policy.yml"
```

Comment:
Could we not just use dictictionaries in the test? No need to create extra files imo.

Reply:
I only did that because I didn't want to make the function too big.

Reply:
I'd like to know the best practice for this, though. Do we create new files only when multiple test cases might need them?

Reply:
It's more about gut feeling I'd say. But the thing that we only do it if multiple test cases need it seems like a good rule of thumb to me 👍